src/Components/Aspire.Azure.Messaging.EventHubs/README.md

```diff
@@ -0,0 +1,145 @@
+# Aspire.Azure.Messaging.ServiceBus
+
+Registers a [ServiceBusClient](https://learn.microsoft.com/dotnet/api/azure.messaging.servicebus.servicebusclient) in the DI container for connecting to Azure Service Bus.
```

Comment:
Looks like this is all still Service Bus content, so skipping for now.

Reply:
Yep, copypasta from the ASB bits. I'll update soon.